lib/proxy.js

```diff
@@ -610,7 +622,7 @@ ReverseProxy.prototype._getTarget = function (src, req) {
       // remove prefix from src
       //
       req._url = url; // save original url
-      req.url = url.substr(pathname.length) || '/';
+      req.url = url.substr(pathname.length) || '';
```

Comment:
why removing the '/' ?

Reply:
the extra `/` cause 404 error.  `test/test_onrequest.js` fails.